openedx/core/djangoapps/discussions/models.py

```diff
@@ -55,85 +55,158 @@ class Features(Enum):
     WCAG_2_1 = 'wcag-2.1'
     WCAG_2_0_SUPPORT = 'wcag-2.0-support'
 
+class LINKS(Enum):
```

Comment:
follow python conventions for class names. https://www.python.org/dev/peps/pep-0008/#class-names
name should be more meaningful. something like `ProviderDocumentationLinks`

Reply:
removed